src/app/models/form.server.model.ts

```diff
@@ -596,6 +596,24 @@ const compileFormModel = (db: Mongoose): IFormModel => {
     )
   }
 
+  // Deletes specified form logic.
+  FormSchema.statics.deleteFormLogic = async function (
+    this: IFormModel,
+    formId: string,
+    logicId: string,
+  ): Promise<IFormSchema | null> {
+    return this.findByIdAndUpdate(
+      mongoose.Types.ObjectId(formId),
```

Comment:
do not need to cast, mongoose will automatically cast for you; `string` type is fine